source/users.rst

```diff
@@ -3,6 +3,27 @@
 Users
 ==============================
 
+User accounts within the Cluster can be divided into two different groups:
+
+* *application-level users*: the unprivileged user accounts,
+* *system-level users*: the accounts needed to automate the cluster deployment
+  and management tasks.
+
+.. _users.system-users:
+
+`System Users <users.html#system-users>`_
+-------------------------------------------
+
+Credentials for system users are stored as a `Kubernetes Secrets <https://kubernetes.io/docs/concepts/configuration/secret/>`_ object.
+The Operator requires to be deployed before PostgreSQL Cluster is
+started. The name of the required secrets (``cluster1-users-secrets`` by default)
```

Comment:
what is the autogenerated name? Is it always cluster1? Or it is <clusterName> ? I would expect the latter.

Reply:
@fiowro, @nonemax tells me that default name is <clusterName>-users. So it would be cluster1-users.
I also created this PR to ensure that by default it is cluster1-users: https://github.com/percona/percona-postgresql-operator/pull/175